editor/src/components/editor/store/collaborative-editing.ts

```diff
@@ -717,3 +717,11 @@ export function useAllowedToEditProject(): boolean {
     'useAllowedToEditProject',
   )
 }
+
+export function useIsMyProject(): boolean {
```

Comment:
Does this clash with the baton passing work? I'm thinking that a lot of the places using this check should be instead using `useAllowedToEditProject`?

Reply:
I don't think it clashes, but it definitely makes it more granular. My take would be to wait that there are no baton-opened PRs to revisit and update